packages/f-content-cards/src/components/cardTemplates/SkeletonLoader.vue

```diff
@@ -3,8 +3,8 @@
         <template v-for="(skeletons, index) in count">
             <div
                 :key="index"
-                :class="$style['c-skeletonLoader-card']"
-                data-test-id="contentCard-skeletonLoader"
+                :class="[$style['c-skeletonLoader-card'], { [$style['c-skeletonLoader-card--promo']]: type === 'promo' }]"
```

Comment:
If you're doing the validation as below, you could include the type here for simplification:
```suggestion
                :class="[$style['c-skeletonLoader-card'], $style[`c-skeletonLoader-card--${type}`]]"
```